package.json

```diff
@@ -25,7 +25,8 @@
     "dev": "turbo run dev",
     "postinstall": "husky install",
     "prepack": "pinst --disable",
-    "postpack": "pinst --enable"
+    "postpack": "pinst --enable",
+    "test:system": "turbo run test:system"
```

Comment:
just a question - is `system` a widely used term for browser e2e tests? Or will this include other things in future across the repo?

Reply:
In our JE repos, we refer to these as system tests. You can find more info here. I just wanted to mainly keep it consistent with JE practice - https://www.guru99.com/system-testing.html

Where as E2E is reserved for more 'user journey', order-flow type tests in JE applications.